djstripe/templates/djstripe/subscribe.html

```diff
@@ -0,0 +1,51 @@
+{% extends "djstripe/base.html" %}
+{% load static djstripe_tags %}
+
+{% block title %}Choose a Subscription{% endblock title %}
+
+{% block content %}
+{{ block.super }}
+<ul class="breadcrumb">
+  <li><a href="{% url 'djstripe:account' %}">Home</a></li>
+  <li class="active">Subscription</li>
+</ul>
+<h2>Choose a Subscription</h2>
+
+{% if error %}
+    <div class="alert alert-error">{{ error }}</div>
+{% endif %}
+{% if view.error %}
+    <div class="alert alert-error">{{ view.error }}</div>
+{% endif %}
+
+<div class="row">
+    {% for plan in PLAN_LIST %}
+      {% with plan_count=PLAN_LIST|length %}
+        <div class="col-xs-{{ 12|djdiv:plan_count|floatformat }}">
+      {% endwith %}
+            {% if customer.current_subscription.plan == plan.plan and customer.current_subscription.status != CurrentSubscription.STATUS_CANCELLED %}
+            <div class="btn btn-default disabled">
+            {% else %}
+            <a class="btn btn-primary" href="{% url 'djstripe:confirm' plan=plan.stripe_plan_id %}" role="button">
```

Comment:
Base on my tests, you will need to replace plan.stripe_plan_id by plan.plan, e.g.:

    <a class="btn btn-primary" href="{% url 'djstripe:confirm' plan=plan.plan %}" role="button">